src/main/java/jenkins/plugins/openstack/compute/ServerScope.java

```diff
@@ -54,6 +54,9 @@
     protected final @Nonnull String specifier;
 
     public static ServerScope parse(String scope) throws IllegalArgumentException {
+        if (scope.startsWith("unlimited")) {
+            return new Unlimited();
```

Comment:
That is fine. It is just not needed to keep re-instantiating it if it has no state - it can be shared among use usages.